pkg/utils/sysinfo_test.go

```diff
@@ -0,0 +1,41 @@
+package utils_test
+
+import (
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/ansible/receptor/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetSysCPUCount(t *testing.T) {
+	got := utils.GetSysCPUCount()
+	assert.Positive(t, got)
+
+	commandOutput, err1 := exec.Command("nproc").CombinedOutput()
+
+	assert.Nilf(t, err1, "Unable to get nproc output, output = %s, err = %s", string(commandOutput), err1)
+
+	commandOutputWithout := strings.TrimSpace(string(commandOutput))
+	want, err2 := strconv.Atoi(commandOutputWithout)
+	assert.Nilf(t, err2, "Unable to convert nproc output %s, err = %s", string(commandOutput), err2)
+
+	assert.Equal(t, got, want)
+}
+
+func TestGetSysMemoryMiB(t *testing.T) {
+	got := utils.GetSysMemoryMiB()
+	assert.Positive(t, got)
+
+	commandOutput, err1 := exec.Command("sed", "-n", "s/^MemTotal:[[:space:]]*\\([[:digit:]]*\\).*/\\1/p", "/proc/meminfo").CombinedOutput()
```

Comment:
I'm assuming this only works on Linux right? I'm not sure if we've broken anything else, but Receptor originally worked on macOS / Linux / Windows. I personally care less about Windows, but I _do_ work with it on my Mac so I really hope we can avoid platform specific stuff in the tests.

Reply:
added check for OS for each